carl/views.py

```diff
@@ -75,3 +77,24 @@ def default(self, obj):
 
     return jsonify(settings)
 
+
+@base_blueprint.route('/process/<int:patient_id>')
+def process(patient_id):
+    from carl.logic.copd import COPD_VALUESET_URI, CNICS_COPD_coding, patient_has, persist_resource
+    from carl.modules.codeableconcept import CodeableConcept
+    from carl.modules.condition import Condition
+    from carl.modules.patient import Patient
+
+    current_app.logger.debug(f"launch process/{patient_id}")
+    positive_codings = patient_has(patient_id=patient_id, value_set_uri=COPD_VALUESET_URI)
+    results = {
+        "patient_id": patient_id,
+        "COPD codings found": len(positive_codings) > 0}
+    if positive_codings:
+        condition = Condition()
+        condition.code = CodeableConcept(CNICS_COPD_coding)
+        condition.subject = Patient(patient_id)
+        response = persist_resource(resource=condition)
+        results['condition'] = response
+
+    return results
```

Comment:
minor nitpick; flipped the check to exit early
```suggestion
    if not positive_codings:
        return results

    condition = Condition()
    condition.code = CodeableConcept(CNICS_COPD_coding)
    condition.subject = Patient(patient_id)
    response = persist_resource(resource=condition)
    results['condition'] = response

    return results
```